Update cloud_providers.md (#2866)
recognai/rubrix: docs/_source/getting_started/installation/deployments/cloud_providers.md

argilla-aws
```

> Available ami depends on region. The provided ami is available for eu-west regions and is an ubuntu-16.04-server image. 

Reviewer: Since here's the first occurrence of `ami`, I'll update it to the following:

```suggestion
Available Amazon Machine Instance (AMI) depends on region. The provided AMI is available for eu-west regions and is an ubuntu-16.04-server image.
```

> To find available recent images, go to the [AWS AMI Marketplace](https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#LaunchInstances:) and filter by `ubuntu`

Reviewer: And the URL here should be region-agnostic

```suggestion
To find available recent images, go to the [AWS AMI Marketplace](https://console.aws.amazon.com/ec2/home), choose "Launch instance". and filter by `ubuntu`
```

(don't forget to choose your targeted region).

> If you already have multiple instances and VPC in the targeted region, creating a new VPC before creating the argilla instance is recommended.

Reviewer: ```suggestion
If you already have multiple instances and VPC in the targeted region, creating a new VPC before creating the Argilla instance is recommended.
```
